### ISSUE TITLE
SFTP Support

### ISSUE THREAD

**Issue by Vanhecke** [cryptomator/cryptomator-ios#99]
It would be great if this app supported SFTP.
That would offer great synergy with the mountain duck app.

Thanks

**Comment by TCB13** [comment]
Yeah, just please add this feature. And when I say SFTP I'm really referring to SSH File Transfer Protocol, not some other hackish FTP-TLS.

**Comment by LeadManPL** [comment]
Any news on this one?

**Comment by Jwink3101** [comment]
I just want to add another vote for this! It will allow access to many self-hosted cloud solutions (e.g. Syncthing, Unison, etc) that don’t offer a WebDAV interface

**Comment by dexinnet** [comment]
Would be great, makes setup & use of custom hosts simpler
